server/auth_function/lambda_function.py

```diff
@@ -74,7 +77,10 @@ def lambda_handler(event: event_type.Event, context: Any) -> event_type.Event:
 
     LOGGER.debug(f"context: {context}")
 
+    LOGGER.info(f"event: {event}")
```

Comment:
I'm not sure if we have any concern about the privacy data from this event data, but I feel this can help us debug if there is any issue in the future.

Reply:
I am not sure either but feel might be okay to leave it ".info" there for a while since we currently starting to have active production apps users with different logins, might need this to debug.
I think preferably to remove this later and instead to figure out how to enable debug mode in production in future.
Thanks.

Reply:
We already capture the user's information in Cognito, so logging it doesn't significantly change the privacy risk. But maybe it should be debug level.

Reply:
(Add a ticket in the backlog for enabling debug mode in production.)

Reply:
yeah.... I was not sure how we can quickly enable debug mode in prod, okay I'll change this to debug log, and create a new ticket for enabling debug mode in production, thanks!